metadata.json

```diff
@@ -20,7 +20,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.4.0"
+      "version_requirement": ">= 4.5.0"
```

Comment:
Shouldn't we bump the module version as well? We usually do that at release time, but I think the new incompatibility with Puppet 3 makes it worth doing in this branch.